Новая должность: Бриг Офицер

Robustin3 opened the pull request:
## What Does This PR Do
Добавление новой должности для охраны.

## Why It's Good For The Game
Смотрителю будет легче, всегда есть под рукой офицер, которого можно послать в пермабриг, а сам открывать аеги, или приказать посадить кого-то в камеру, пока сам в карго.

## Images of changes
![image](https://user-images.githubusercontent.com/109764045/180863659-e2043d97-a2ed-499c-9bc9-35d96c966151.png)


## Changelog
:cl:
add: brig officer job
brig officer's cloth, pda
brig officer's skin for plasmamen
brig officer's own closet
/:cl:

Robustin3 committed:
новая должность: бриг офицер

Варден младший, казалось бы бесполезен, однако когда не хватает рук смотрителю, становится не заменимым. свой шкафчик, одежда.

Robustin3 committed:
Update delta.dmm

а это шкафчик, положение раундстарт бригофицера

Robustin3 committed:
Merge branch 'brig-officer' into master220

Blundir committed:
Добавление спрайтов пда, шкафа, фиксы, регулировки

Blundir committed:
Исправление спрайтов плазмаменов

Robustin3 committed:
смена позиции шкафа

название говорит само за себя

Robustin3 committed:
Merge branch 'master220' of https://github.com/Robustin3/Paradise into master220

Robustin3 committed:
опять шкафчик

труба что-то убирается. теперь наверное не должна

Robustin3 committed:
опять шкафчик 2

труба что-то убирается. теперь наверное не должна

Robustin3 committed:
ШКАФЧИК УЖЕ НАВЕРНЯКА

НУ ДОЛЖЕНННННН

Robustin3 committed:
Последние штрихи

мелкая правка, шапка в лоудауте, берет бригофицера в лоудауте для хоса.

BeebBeebBoob commented:
Такой ПР не пойдет совсем. Тебе надо было открывать и изменять карты, которые dmm файлы в спец программах как strongdmm или ещё одна которую не помню.

VasilevAY commented:
Считаю, что данную предложку не имеет смысла мерджить до тех пор, пока не будет пройдена отдельная предложка по СРП для бриг офицера. В том виде, в котором она есть (см. оригинальную предложку в дс), она не может быть принята, на мой взгляд

PhantornRU commented:
Без клоуна-смотрителя, предложка не является полноценной

BeebBeebBoob commented on .gitignore:
```diff
@@ -60,3 +60,4 @@ stddef.dm
 
 # Ignore compiled linux libs in the root folder, e.g. librust_g.so
 /*.so
+paradise.dme
```
Это вообще не надо трогать.

BeebBeebBoob replied:
```suggestion

```
всё ещё не надо трогать .gitignore. Это не в наших силах, это в силах азиза делать такие здесь изменения если потребуются.

larentoun commented:
А засквашить не судьба? Так и жду увидеть 10 коммитов "шкафчик"

VasilevAY commented:
> Ладно, будет вам карта delta.dmm в strongdmm... Только не сейчас.

а как это относится к данной предложке?

larentoun commented:
> Ладно, будет вам карта delta.dmm в strongdmm... Только не сейчас.

И на кибериаду тоже нужно

Murguletto commented:
А черт, это я ст твинка пишу

Robustin3 committed:
Шкафчик на Кибериаде

Теперь шкафчик на кибериаде, а еще дельта и кибериада изменены с помощью strongdmm

Robustin3 commented:
ну как бы дельта и кибериада, шкфчики есть, карты измменены в strongdmm.

BeebBeebBoob commented:
тебе надо было изначально это сделать. Теперь тебе надо удалить все карты которые ты изменил, потом скачать с гитхаба самую актуальную от азиза. И всё ещё...
![image](https://user-images.githubusercontent.com/32985153/181194527-994c3599-564d-4d49-8bdc-970a17a3a987.png)
слишком много

BeebBeebBoob commented on paradise.dme:
```diff
@@ -12,6 +12,7 @@
 // BEGIN_INCLUDE
 #include "_maps\__MAP_DEFINES.dm"
 #include "_maps\delta.dm"
+#include "_maps\map_files\Delta\delta.dmm"
```
```suggestion

```
здесь идут только файлы .dm. не карты, не звуки, ничего кроме dm

Robustin3 replied:
я вообще в первый раз, если надо что-то изменить, то без проблем

Robustin3 committed:
Еще штрихи

Требования бип бип боба выполнены.

Robustin3 committed:
мини-фикс

при выборе всех доступов не выбирался бригофицерский, исправлено

Robustin3 committed:
очепятка

эээ ладно
